flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/ExecutionCheckpointingOptions.java

```diff
@@ -250,6 +251,10 @@ public class ExecutionCheckpointingOptions {
                                             "Forces unaligned checkpoints, particularly allowing them for iterative jobs.")
                                     .build());
 
+    /**
+     * @Deprecated Use {@link StateRecoveryOptions#CHECKPOINT_ID_OF_IGNORED_IN_FLIGHT_DATA} instead.
```

Comment:
It's better to use lowercase @deprecated tag.
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@deprecated